src/components/PrivacyPolicy/PspPrivacyInfo.tsx

```diff
@@ -0,0 +1,47 @@
+import React from "react";
+import { Link, Typography } from "@mui/material";
+import { Trans } from "react-i18next";
+
+const PrivacyLink = (props: {
+  href: string;
+  children?: string;
+  rel?: string;
+}) => {
+  const { href, children, rel = "noreferrer" } = props;
+  return (
+    <Link
+      href={href}
+      target="_blank"
+      rel={rel}
+      style={{ fontWeight: 600, textDecoration: "none" }}
+    >
+      {children}
+    </Link>
+  );
+};
+
+interface PspPrivacyInfoProps {
+  termsLink: string;
+  privacyLink: string;
+  pspName?: string;
```

Comment:
```suggestion
  pspName: string;
```

According to the flow, this banner shouldn't be visible if there’s no PSP selected, right? The translation string is made up of static text and the PSP name.
If this assumption is correct, we should probably make pspName a required parameter and update the main logic to only render this component when a PSP is selected.
What do you think?

Reply:
I've updated logic